elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -136,13 +138,17 @@ public void registerTransformers(SubsystemTransformerRegistration registration)
 
     private static void from15_1(ChainedTransformationDescriptionBuilder chainedBuilder) {
         ResourceTransformationDescriptionBuilder builder = chainedBuilder.createBuilder(ELYTRON_15_1_0, ELYTRON_15_0_0);
-
+        builder.addChildResource(PathElement.pathElement(FILESYSTEM_REALM))
+                .getAttributeBuilder()
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, CREDENTIAL_STORE)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, SECRET_KEY)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, CREDENTIAL_STORE)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, SECRET_KEY);
     }
 
     private static void from15(ChainedTransformationDescriptionBuilder chainedBuilder) {
         ResourceTransformationDescriptionBuilder builder = chainedBuilder.createBuilder(ELYTRON_15_0_0, ELYTRON_14_0_0);
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.JAAS_REALM));
-
```

Comment:
Would be good to remove the whitespace here.

Reply:
Fixed